src/milia/api/async_export.cljc

```diff
@@ -19,9 +20,10 @@
    to explicity pass a question-mark or ampersand depending on whether this is
    the first or a subsequenty query parameter."
   [& s]
-  (join (if-let [temp-token (:temp-token *credentials*)]
+  (let [temp-token (:temp-token *credentials*)]
+  (join (if (is-not-null? temp-token)
```

Comment:
this should be indented, since it's w/i the `let`

Reply:
Right nice catch, lemme fix that.